_posts/2016-12-06-future.md

```diff
@@ -0,0 +1,444 @@
+- [@maxiloc](https://github.com/maxiloc) added search functionality via Algolia via [#977](https://github.com/babel/babel.github.io/pull/977)
+
+and added some new collaborators: 
+
+- [@STRML](https://github.com/STRML): Added discourse to all github pages via [#875](https://github.com/babel/babel.github.io/pull/875)
+- [@xtuc](https://github.com/xtuc): Added support for reading the README from the babel repo so we don't have to sync 2 copies of docs via [#990](https://github.com/babel/babel.github.io/pull/990)
+- [@fredericmarx](https://github.com/fredericmarx): Added support for a copy to clipboard button for code snippets via [#998](https://github.com/babel/babel.github.io/pull/998)
+- [@seedofjoy](https://github.com/seedofjoy): Added a resize ability for the REPL via [#1003](https://github.com/babel/babel.github.io/pull/1003)
+
+Some Ideas
+
+- All plugins should have examples. Can also embed [RunKit](https://runkit.com/home) or the REPL.
+- Update FAQ with common errors
+- API docs / make [babel-handbook](https://github.com/thejameskyle/babel-handbook) better
+
+Relevant Issues:
+
+- [Reorganizing the docs layout](https://github.com/babel/babel.github.io/issues/930)
+- [Showcase community plugins](https://github.com/babel/babel.github.io/issues/831)
+- [Adding a resources page](https://github.com/babel/babel.github.io/issues/27)
+- [Learn ES2015: teach others about ES2015+](https://github.com/babel/babel.github.io/issues/994)
+- [REPL: adding example dropdowns](https://github.com/babel/babel.github.io/issues/59)
+- [REPL: supporting all options](https://github.com/babel/babel.github.io/issues/92)
+- [REPL: allow the REPL to import any community plugin from npm](https://github.com/babel/babel.github.io/issues/858)
+- [REPL: Use ASTexplorer](https://github.com/fkling/astexplorer/issues/70)
+- [ASTexplorer: Real Time collaboration](https://github.com/fkling/astexplorer/issues/166)
+
+## Future
+
+These are all things that can be changed or dropped. Some are already in the works and others need a proper discussion/champion.
+
+### [Plugin API Changes](https://github.com/babel/notes/blob/master/2016-08/august-01.md#potential-api-changes-for-traversal)
+
+There is a lot of confusion around how plugins/presets interact regarding ordering. This results in bugs and issues with the config which require users to place plugins before/after others in a non-intuitive way.
+
+This is a fundamental change to the core of Babel which will take some work to figure out.
+
+### Versioning
+
+Since Babel 6 we've used a ["fixed" mode of versioning](https://github.com/lerna/lerna#how-it-works) via Lerna. This is what allows us to release multiple packages at the same time all under the same version (if that package changes). This is nice because you don't have to manually set a version for each package but everything moves together. The only time this is a pain is when one on the packages makes a breaking change: then every package also will bump major versions.
```

Comment:
>  The only time this is a pain is when one on the packages makes a breaking change: then every package also will bump major versions

I see this as a feature rather than a bug :) Having one version number for Babel as a whole simplifies things a lot.